guides/hack/06-classes/18-constants.md

```diff
@@ -9,17 +10,24 @@ class Automobile {
 
 <<__EntryPoint>>
 function main(): void {
-  $col = Automobile::DEFAULT_COLOR;
+  $col = Automobile::DEFAULT_COLOR; // or: $col = self::DEFAULT_COLOR;
   echo "\$col = $col\n";
 }
 ```
 
-If a class constant's type is omitted, it is inferred from the initializer, which must be present. In this case, that type is `string`.
+## Visibility
+Class constants have public visibility and can not be declared as `public`, `protected`, or `private`.
```

Comment:
Class constants are always public, and can not be explicitly declared as `public`, `protected`, or `private`.